service/initialise.go

```diff
@@ -3,26 +3,30 @@ package service
 import (
 	"net/http"
 
+	"github.com/ONSdigital/dp-api-clients-go/v2/dataset"
```

Comment:
I don't understand why the service needs to be initialising the dataset client, the script has little to do with the service. The only thing they may share is the configurations; if the script needs extra config then it should extend the service config.

Reply:
sorry didn't understand this client, the dataset client is needed to call the dataset client apis .Please note this is ```dp-api-clients-go``` not the actual dataset service.

Reply:
This is fixed in update 1.